scio-test/src/test/scala/com/spotify/scio/values/PairSCollectionFunctionsTest.scala

```diff
@@ -546,14 +544,29 @@ class PairSCollectionFunctionsTest extends PipelineSpec {
   }
 
   it should "support topByKey()" in {
+    runWithContext { sc =>
+      val p = sc.parallelize(Seq(("a", 1), ("b", 11), ("b", 12), ("c", 21), ("c", 22), ("c", 23)))
+      val r1 = p.topByKey(1)
+      val r2 = p.topByKey(1, Ordering.by(-_))
+      r1 should
+        containInAnyOrder(Seq(("a", Iterable(1)), ("b", Iterable(12)), ("c", Iterable(23))))
+      r2 should
+        containInAnyOrder(Seq(("a", Iterable(1)), ("b", Iterable(11)), ("c", Iterable(21))))
+    }
+  }
+
+  // TODO explain this test case
+  // Kept for backward compat
+  it should "support topByKey() with JavaWrappedIterables" in {
```

Comment:
You can remove this case if we remove `def iterable()` instead of deprecate?

Reply:
yes, we can remove this. I was thinking of keeping backward compat for those who hacked around, but i don't want to, so I'll remove this.